pontoon/teams/static/js/permissions.js

```diff
@@ -2,13 +2,16 @@ $(function() {
   var container = $('#main .container');
 
   function inputHidden(name, value) {
-    return $('<input type="hidden" name="' + name + '" value="' + value + '">');
+    return $('<input class="permissions-form-item" type="hidden" name="' + name + '" value="' + value + '">');
   }
 
   container.on('click', '#permissions-form .save', function(e) {
     e.preventDefault();
     var $form = $('#permissions-form');
 
+    // Remove stale items.
+    $form.find('input.permissions-form-item').remove();
```

Comment:
Is this different from `$('input.permissions-form-item', $form).remove();`? Note that this is just me being curious, it's the first time I see it written like this.

Reply:
Same thing. :)

Reply:
I thought it may be a little faster than `$()` in this case. However, maybe your notation will be easier to read and more explicit.